telemetry/gparser/parser.py

```diff
@@ -243,13 +245,153 @@ def get_payload_parsed(self):
+                                        insert_param_name = param_name + "="
+                                        param_list.insert(0, insert_param_name)
+                                        param_list_string = "   \n".join(param_list)
+                                        # Update param in param_separate list
+                                        param_separate[index] = param_list_string 
+                        # Compile final parameter details
+                        param_separate.insert(0,"**Parameters:**")
+                        param_display = "\n  - ".join(param_separate)
+
+                        # Get failure tag content
+                        failure_text = failure.text
+                        fail_content_lines = failure_text.splitlines()
+                        exc_param_value = ""
+                        # Get exception statement with parameter names
+                        for item in fail_content_lines[::-1]:
+                            if len(item) > 0:
+                                if item[0] == ">":
+                                    exc_param_value = item[1:].lstrip()
+                                    break                        
+                                                                     
+                        test_name_function = ""
+                        test_function_module = ""
+                        exctype_message = ""
+                        # Iterate through each property in the properties tag
+                        for prop in properties.findall("property"):
+                            # Get the property name and value
+                            prop_name = prop.get("name")
+                            prop_value = prop.get("value")
+                            if prop_name == "exception_type_and_message":                                                             
+                                prop_list = prop_value.splitlines() 
+                                new_props = prop_value
+                                prop_list_updated = []
+                                # Check if exception and message has mutiple lines  
+                                if len(prop_list) > 1:
+                                    for props in prop_list:
+                                        if len(props) > 0:
+                                            # Remove leading spaces
+                                            prop_strip = props.lstrip()
+                                            prop_list_updated.append(prop_strip)                                        
+                                if len(prop_list_updated) > 0:
+                                    new_props = " ".join(prop_list_updated)                                            
+                                # Combine exception type, message, and parameters        
+                                exctype_message =  "\n" + "  " + new_props + " ( " + exc_param_value + " )"
```

Comment:
What's an example of a property value that satisfies this condition and requires such parsing?